server.js

```diff
@@ -1,39 +1,113 @@
+import dotenv from 'dotenv'
+
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import booksData from './data/books.json'
+
+dotenv.config()
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const bookSchema = new mongoose.Schema({
+  bookID: Number,
+  title: String, 
+  authors: String,
+  average_rating: Number,
+  isbn: String, 
+  isbn13: String,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+const Book = mongoose.model('Book', bookSchema)
+
+if (process.env.RESET_DB) {
+  const seedDataBase = async () => {
+    await Book.deleteMany()
+
+    booksData.forEach((bookData) => {
+      new Book(bookData).save()
+    })
+  }
+  seedDataBase()
+}
+
 const port = process.env.PORT || 8080
-const app = express()
+const app = express() 
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: /books, /books?title=titanic, /books?languages_code=eng, /books/isbn/076790818X, /books/id/609bcffa120fc13f88868db7')
+})
+
+// All books
+// http://localhost:8080/books 
+// Find title or language
+// http://localhost:8080/books?title=titanic
+// http://localhost:8080/books?languages_code=eng
+app.get('/books', async (req, res) => {
+  // eslint-disable-next-line camelcase
+  const { title, language_code } = req.query
+
+  if (title) {
+    const bookInfo = await Book.find({
+      title: {
+        $regex: new RegExp(title, 'i')
+      }    
+    })
+    res.json(bookInfo)
+  }
+
+  // eslint-disable-next-line camelcase
```

Comment:
When I first tried to enter a different language code than 'eng' I received all books instead. Then I noticed that your param is called language_code and in all other places (comments, README and in the '/' endpoint) you have written it as languages_code (and that's why nothing happened when I typed anything)  🐱‍👤